db/migrate/20200428143250_add_primary_key_to_rules_references.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrimaryKeyToRulesReferences < ActiveRecord::Migration[5.2]
+  def change
+    add_column :rule_references_rules, :id, :primary_key
```

Comment:
@dLobatog what does this do?

Reply:
This allows us to delete the duplicates. Without it, you can find the duplicates, but you can't delete them - think about:
```
DELETE FROM "rule_references_rules" WHERE rule_id = foo AND rule_reference_id = bar 
```
or any statement you could construct would delete *all* of the records for a rule_id/rule_reference_id pair. Without a primary key, you can delete either by rule_id (all of the records for a same rule_id), rule_reference_id (same) or both (same).

Reply:
Ok, that makes sense. Thanks for the explanation! :+1: